src/submission/parseErrors/index.jsx

```diff
@@ -23,17 +27,14 @@ export const renderTSErrors = ({ transmittalSheetErrors }) => {
       <thead>
         <tr>
           <th>Row</th>
-          <th>Transmittal Sheet Errors</th>
+          <th>TS Data Field</th>
+          <th>Found Value</th>
+          <th>Valid Value</th>
         </tr>
       </thead>
       <tbody>
-        {transmittalSheetErrors.map((tsError, i) => {
-          return (
-            <tr key={i}>
-              <td>1</td>
-              <td>{tsError}</td>
-            </tr>
-          )
+        {tsErrorMessages.map((tsError) => {
```

Comment:
Lines 36-38 can just be the following:
```suggestion
        {tsErrorMessages}
```